guides/hack/11-built-in-types/35-enum-class.md

```diff
@@ -52,19 +69,21 @@ enum class Names : IHasName {
 hhvm.hack.lang.enable_enum_classes=1
 ```
 
-## Declaring a new enum class
+## Accessing enum class constants
 
-Enum classes are syntactically different from existing enum types, as they require:
+Once declared, enum class constants are accessed like normal enum constants using the `::` operator: `Names::Hello`, `Names::Bar`, ...
 
-* the `enum class` keyword rather than the `enum` keyword
-* that each constant is annotated with its precise type, as in `HasName Hello = ...`
+### Control over enum class constants
 
-Once declared, enum class constants are accessed like normal enum constants using the `::` operator: `Names::Hello`, `Names::Bar`, ...
+Using [coeffects](../contexts-and-capabilities/introduction.md), you can have control over which expressions are allowed as enum class constants.
+
+By default, all enum classes are under the `write_props` context. It is not possible to override this explicitly. The only work around must be a temporary one involving `HH_FIXME`s.
```

Comment:
What is 'one' here? We should either explain it clearly (with an example) and say to avoid it and why, or just remove the mention entirely.